docs/mllib-statistics.md

```diff
@@ -521,7 +521,7 @@ val sc: SparkContext = ...
 
 // Generate a random double RDD that contains 1 million i.i.d. values drawn from the
 // standard normal distribution `N(0, 1)`, evenly distributed in 10 partitions.
-val u = normalRDD(sc, 1000000L, 10)
+val u = RandomRDDs.normalRDD(sc, 1000000L, 10)
```

Comment:
The symbol is already imported though; it works without the class name. The example should work.